workers/generic-worker/main.go

```diff
@@ -1249,5 +1269,6 @@ func exitOnError(exitCode ExitCode, err error, logMessage string, args ...interf
 	log.Printf(logMessage, args...)
 	log.Printf("Root cause: %v", err)
 	log.Printf("%#v (%T)", err, err)
+	errorreport.Send(WorkerRunnerProtocol, err, debugInfo)
```

Comment:
This error report should include `exitCode`, `err`, `logMessage` and `args`, not just `err`, since a lot of important context is lost if you only include `err`.